vcd_cli/catalog.py

```diff
@@ -114,6 +114,27 @@ def info(ctx, catalog_name, item_name):
     except Exception as e:
         stderr(e, ctx)
 
+
+@catalog.command(short_help='catalog control access details')
+@click.pass_context
+@click.argument('catalog-name',
+                metavar='<catalog-name>',
+                required=True)
+def control_access(ctx, catalog_name):
+    try:
+        client = ctx.obj['client']
+        org_name = ctx.obj['profiles'].get('org')
+        in_use_org_href = ctx.obj['profiles'].get('org_href')
+        org = Org(client, in_use_org_href, org_name == 'System')
+        control_access = org.get_catalog_access_control_settings(catalog_name)
+        stdout('Access Settings for catalog :' + catalog_name)
+        stdout(control_access.get('AccessSettings'), ctx, show_id=True)
+        del control_access['AccessSettings']
```

Comment:
For my understanding, what is the goal of del here? It is anyway going to go out of scope end of the method which isn't too far?

Reply:
The goal here is to have the output printed in a presentable way. If control_access object is printed directly, it gives out the access_setting information in a list of map that is difficult to read. So, the manipulation here is to print the access_setting maps and then remove that from the control_access object and and print the rest. If you refer to the output of the command above, you would get the gist of it.

Reply:
Understood, thanks.